test/runtests.jl

```diff
@@ -1,8 +1,11 @@
-using Base.Test
 using SortingAlgorithms
-using Compat
+using Compat, Compat.Test
 using StatsBase
 
+if !isdefined(Base, :invpermute!)
+    invpermute! = ipermute!
```

Comment:
`const`

Reply:
Actually `invpermute!` is available in Compat 0.44.0, so this should be unnecessary.